crates/compiler/load_internal/src/file.rs

```diff
@@ -666,7 +668,7 @@ struct ModuleHeader<'a> {
     is_root_module: bool,
     exposed_ident_ids: IdentIds,
     deps_by_name: MutMap<PQModuleName<'a>, ModuleId>,
-    packages: MutMap<&'a str, PackagePath<'a>>,
+    packages: MutMap<&'a str, PackageName<'a>>,
```

Comment:
Wait, this diff shows that we already had something for this?
I guess there were some other PRs building up to this, is that it?

Reply:
Exactly!